controllers/handlers/kubevirt.go

```diff
@@ -1017,3 +1048,23 @@ func getLabelPatch(dest, src map[string]string) ([]byte, error) {
 
 	return json.Marshal(patches)
 }
+
+func passtNetworkBinding(sidecarImage string) kubevirtcorev1.InterfaceBindingPlugin {
+	return kubevirtcorev1.InterfaceBindingPlugin{
+		NetworkAttachmentDefinition: networkAttachmentDefinition,
+		SidecarImage:                sidecarImage,
+		Migration:                   &kubevirtcorev1.InterfaceBindingMigration{},
+		ComputeResourceOverhead: &kubevirtcorev1.ResourceRequirementsWithoutClaims{
+			Requests: corev1.ResourceList{
+				corev1.ResourceMemory: passtResourceMemory,
+			},
+		},
+	}
+}
+
+func passtImageOnceFn() string {
```

Comment:
nit - naming: Not sure why we need the `Fn` suffix, and `Once` describing the "algorithm" within the function, that is not relevant to the user/reader of the function. I would simply call it `getPasstImage` or something like that.